Changelog.md

```diff
@@ -1,5 +1,8 @@
 ## Changes
 
+### 5.1.3
```

Comment:
6.0.0 ? I mean the change is not retro compatible I think

Reply:
It should be 6.1.0. Our 5.x branch is for older version of ReScript.